chainercv/links/model/fpn/faster_rcnn.py

```diff
@@ -26,6 +26,9 @@ class FasterRCNN(chainer.Chain):
         head (Link): A link that has the same interface as
             :class:`~chainercv.links.model.fpn.Head`.
             Please refer to the documentation found there.
+        min_size (int): A preprocessing paramter for :meth:`prepare`. Please
+            refer to a docstring found for :meth:`prepare`.
```

Comment:
How about this?
```
Please refer to :meth:`prepare`.
```
`docstring` will be confusing when it rendered by sphinx.

Reply:
This was done in the same style as the faster-rcnn docs with regards to the param.

Reply:
I see.